widget-v2

andrewlee348 opened the pull request:
feat: added home header

SamueleA reviewed (changes requested):
Looks good!
A few things I noticed:

1. NFTs which aren't squares don't display well. We should have the square be able to display the entirety of the image of an NFT.


![image](https://github.com/user-attachments/assets/9455072e-2580-423a-ac3b-118c79633f73)
![image](https://github.com/user-attachments/assets/bca252d5-9fcb-451a-80c8-b14dcc85f572)
You can test with the following collection (if you DM me I can send you one for testing): https://polygonscan.com/token/0x631998e91476da5b870d741192fc5cbc55f5a52e

2. The explore tab of a collection doesn't seem to be doing anything. Not sure if it's incomplete or buggy. If the former, we can always remove the tab option.
![image](https://github.com/user-attachments/assets/ec457804-67d1-4464-894b-cdf24747aeed)

3. There's a black circle over my gravatar. I'm not sure what's supposed to go there. I'm assuming it's the current network? 
 
![image](https://github.com/user-attachments/assets/f63c8272-20e6-41a4-bc68-e5ee6f86367d)

4. I've managed to break the app, but I'm a bit unsure what is causing the issue.
Here are the steps I did:
a. in the home of the demo app, click on universal wallet and connect with a sequence wallet through facebook or google
b. open the widget, click the walet icon and click on "+ add wallet"
c. The app will crash.
d. Then, clear application data, go to the home page of the demo application, click universal wallet. The app will crash.

In my case, I was already connected to the app on localhost:4444 so it didn't go through the connect modal for universal. In other words, the two scenarios that need second view are clicking on "+ add wallet" when connected with a sequence universal wallet (not waas) and clicking the "universal wallets" option in the demo app with all application data cleared.

andrewlee348 commented:
1. I've gone and reverted my change to fill the entire tile. You should be able to see the full height of collectibles now.
![image](https://github.com/user-attachments/assets/76bfc18c-7e09-48df-8b05-203c3ef87176)
![image](https://github.com/user-attachments/assets/cb9e067e-5ef1-4108-a39e-ee495f2eb26c)
2. The explore tab is incomplete as the designs didn't delve into what that would contain. I've gone and added a coming soon description for now.
![image](https://github.com/user-attachments/assets/aed1c316-8e09-4f05-ab7f-ba16bee16eca)
3. The black circle is because there isn't a logo for guest in this pr. I've added one in this pr: https://github.com/0xsequence/web-sdk/pull/460 so once thats merged, I can rebase and it will show up.
4. This has to do with the connect modal being updated to add guest logins. There was a missing condition to check whether the connect was waas or universal before instantiating a SequenceWaaS object.
